applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -15,6 +15,11 @@ def __init__(self, Model, settings ):
         default_parameters = KratosMultiphysics.Parameters( """
             {
                 "model_part_name":"PLEASE_CHOOSE_MODEL_PART_NAME",
```

Comment:
```suggestion
                "model_part_name":"",
```
In this way you can check if the model part name has been provided or not  by comparing what you get against an empty string.

Reply:
Alright